### PR TITLE
Sleep time between retries

### PR DESCRIPTION
**Pull request by markettes**
This one implemets:
- Incremental sleep times between retries
- Max sleep time

### CODE REVIEW

**Review by gcaracuel — DISMISSED** [review]
LGTM.
I won't override new flags as default values sounds like very good first pattern to check.

File: liquidator.go [hunk 1]
```diff
@@ -507,7 +510,7 @@ func manageChannelLiquidity(info ManageChannelLiquidityInfo) error {
 
 }
 
-func performSwap(info ManageChannelLiquidityInfo, channel *lnrpc.Channel, swapAmount int64, rule nodeguard.LiquidityRule, span trace.Span, loopdCtx context.Context, retryCounter int, swapAmountTarget int64) error {
+func performSwap(info ManageChannelLiquidityInfo, channel *lnrpc.Channel, swapAmount int64, rule nodeguard.LiquidityRule, span trace.Span, loopdCtx context.Context, retryCounter int, swapAmountTarget int64, sleepBetweenRetries time.Duration) error {
```
Comment (Jossec101) [inline liquidator.go:513]: if `sleepBetweenRetries` is a global, why send it over as a function param?

File: README.md [hunk 2]
```diff
@@ -64,53 +72,71 @@ All the flags can be set as environment variables, with the following format, ex
 - RETRIESBEFOREBACKOFF (optional) : Number of retries before applying backoff to the swap (default: 3)
 - BACKOFFCOEFFICIENT (optional) : Coefficient to apply to the backoff (default: 0.95)
 - BACKOFFLIMIT (optional) : Limit coefficient of the backoff (default: 0.1)
+- SLEEPBETWEENRETRIES (optional) : Base time to sleep between retries (default: 500ms)
+- SLEEPBETWEENRETRIESBACKOFF (optional) : Coefficient to apply to the backoff (default: 1.5)
+- SLEEPMAX (optional) : Maximum time to sleep between retries (default: 60s)
```
Comment (Jossec101) [inline README.md:77]: I think this is not useful, I would reduce code and actually control this via config rather making code more complex.

**Review by Jossec101 — CHANGES_REQUESTED** [review]
Sleep max can be removed, its not worth the code it adds for something trivial

**Comment by gcaracuel** [comment]
I guess this PR has fall behind some others and have not much sense now

**Comment by markettes** [comment]
> I guess this PR has fall behind some others and have not much sense now

Test, I guess so

**Comment by Jossec101** [comment]
Let's close it

**Pull request closed without merging**